app/components/CallCharacter/index.tsx

```diff
@@ -379,6 +391,13 @@ export function CallCharacter({ character }: { character: CharacterType }) {
 
   return (
     <CheckTooBusy>
+      <CallError
+        err={callError}
+        open={callDidError}
```

Comment:
Ah. Maybe rename callDidError to "errorDialogOpen" since that's what this really means.